measurement/util.py

```diff
@@ -114,6 +123,15 @@ def parse_insight_opts():
     parser.add_argument("-v", "--verbose", action="store_true", default=False,
                         help="Print verbose output.")
 
+    parser.add_argument("-f", "--ftrace", action="store_true", default=False,
+                        help="Collect trace info using ftrace. Disabled by default.")
+    parser.add_argument("--ftracepoint",  action="store", default=None,
+                        help="Tracepoint to be traced")
```

Comment:
add period(.) in the end of `ftracepoint ` help message.
Which tracepoint does it currently support?